tests/test_tree.rs

```diff
@@ -49,7 +50,7 @@ fn kv(i: usize) -> Vec<u8> {
 #[test]
 fn it_works() {
     println!("========== initial sets ==========");
-    let t = Tree::new();
+    let t = Arc::new(Tree::new());
```

Comment:
Not sure if that's the root cause, but it looks like this change causes OS X builds to fail.